apidoc/Titanium/UI/View.yml

```diff
@@ -1119,7 +1119,7 @@ properties:
                         startPoint: { x: 50, y: 50 },
                         endPoint: { x: 50, y: 50 },
                         colors: [ 'red', 'blue'],
-                        startRadius: '90%',
+                        startRadius: 90,
```

Comment:
We should set startRadius to `50` instead.
This way, a full circle will just-fit within the view.

Note that setting this to `90` would make the radius larger than the old `90%` setting. On iOS, `100%` equals `sqrt(width^2, height^2) / 2`, which is the distance between the center of the view to one of its corners.